contracts/stake-cw20-reward-distributor/src/contract.rs

```diff
@@ -144,7 +144,7 @@ fn get_distribution_msg(deps: Deps, env: &Env) -> Result<Option<CosmosMsg>, Cont
     let config = CONFIG.load(deps.storage)?;
     let last_payment_block = LAST_PAYMENT_BLOCK.load(deps.storage)?;
     let block_diff = env.block.height - last_payment_block;
-    if block_diff <= 0 {
+    if block_diff == 0 {
```

Comment:
Isn't it possible that `last_payment_block > env.block.height`? Check [this](https://github.com/DA0-DA0/dao-contracts/pull/352#issuecomment-1151090605) for a suggested fix